internal/pkg/cli/svc_deploy.go

```diff
@@ -372,6 +376,13 @@ func (o *deploySvcOpts) manifest() (interface{}, error) {
 }
 
 func (o *deploySvcOpts) runtimeConfig(addonsURL string) (*stack.RuntimeConfig, error) {
+	rc := stack.RuntimeConfig{
+		AddonsTemplateURL: addonsURL,
+		AdditionalTags:    tags.Merge(o.targetApp.Tags, o.resourceTags),
+	}
+	if !o.buildRequired {
+		return &rc, nil
+	}
```

Comment:
nit: inline the structs in the return statements
```suggestion
	if !o.buildRequired {
		return &stack.RuntimeConfig{
			AddonsTemplateURL: addonsURL,
			AdditionalTags:    tags.Merge(o.targetApp.Tags, o.resourceTags),
		}, nil
	}
```